examples/Videos/Sample/ViewController.m

```diff
@@ -55,16 +56,6 @@ - (void)viewWillAppear:(BOOL)animated
   [self.view addSubnode:_rootNode];
 }
 
-- (void)viewDidLayoutSubviews
```

Comment:
It should actually not be in `viewWillAppear:` at all. As the get's called every time the view controller appears. I moved the code to `viewDidLoad`.

To your question. As you can see while creating the root node we set the autoresizing mask to flexible width and height so we don't have to adjust the frame after a layout of the view controller subviews. You can also see it if you set a background color to the root node it expands to the whole view also if rotation is happening.